source/Examples/SmartCache/SmartCache.SettingsRegistry/AssemblyInitializer.cs

```diff
@@ -134,18 +207,19 @@ private static void RequestNewConfigurationMessageHandler(SmartCacheProxy.Reques
 
                     string currentConfigJsonString = JsonSerializer.Serialize(currentConfigDictionary, JsonOptions);
 
-                    // Register an observation.
+                    // Send (register) the observation with the optimizer.

```

Comment:
"send with the optimizer" doesn't really work? with/to?

Reply:
reworked the wording